src/core/Akka.Remote.TestKit/CommandLine.cs

```diff
@@ -26,17 +26,19 @@ namespace Akka.Remote.TestKit
     ///     var testName = CommandLine.GetProperty("multinode.test-method");
     /// </code>
     /// </summary>
-    public class CommandLine
+    public static class CommandLine
     {
-        private static readonly StringDictionary Values;
+        private static readonly StringDictionary Values = new StringDictionary();
 
         static CommandLine()
         {
-            Values = new StringDictionary();
+            Initialize(Environment.GetCommandLineArgs());
```

Comment:
Have to do this, else net472 tests would fail. It seems like static scope in net472 is different than in netcore, static somehow didn't get propagated from the node runner to the xunit test runner.